Tests/TestMessagesViewControllerModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TestMessagesViewControllerModel.swift
+//  MessageKit
+//
+//  Created by Dan Leonard on 8/14/17.
+//  Copyright © 2017 MessageKit. All rights reserved.
+//
```

Comment:
Do you want me to manually just put the license or what would you like done in this case?